src/utils/include/com/amazonaws/kinesis/video/utils/Include.h

```diff
@@ -1375,6 +1375,65 @@ PUBLIC_API SIZE_T getInstrumentedTotalAllocationSize();
 #define RESET_INSTRUMENTED_ALLOCATORS()             resetInstrumentedAllocatorsNoop()
 #endif
 
+//////////////////////////////////////////////////////////////////////////////////////////////////////
+// File logging functionality
+//////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/**
+ * File logger error values starting from 0x41300000
+ */
+#define STATUS_FILE_LOGGER_BASE                                             STATUS_UTILS_BASE + 0x01200000
+#define STATUS_FILE_LOGGER_INDEX_FILE_INVALID_SIZE                          STATUS_SEMAPHORE_BASE + 0x00000001
+
+/**
+ * File based logger limit constants
+ */
+#define MAX_FILE_LOGGER_STRING_BUFFER_SIZE          (3 * 1024 * 1024)
```

Comment:
Perhaps we can bump up the max file logger buffer size to something larger. There is no reason for it to limit the max

Reply:
createFileLogger

Reply:
This is a max and in order to change the behavior and make it not fail the customer needs to change this constant.

Reply:
this is default max, it's configurable in create file logger